Cargo.lock

```diff
@@ -2285,7 +2285,7 @@ dependencies = [
 [[package]]
 name = "outbound-redis"
 version = "0.2.0"
-source = "git+https://github.com/danbugs/spin?branch=danbugs/custom-logging-pipes#d79eaf0cdcd4c71982ea8eb2f97e9c477b2acb3a"
+source = "git+https://github.com/danbugs/spin?branch=danbugs/custom-logging-pipes#635f4d73b25884d27351d1862aae6a58096810bb"
```

Comment:
can you add `rev` to the spin dependency in `Cargo.toml`

Reply:
We are currently using my fork of spin. As I'm updating it kind of constantly while the PR is under review, I've avoided using a rev to save me the trouble of changing it all the time.